hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieUnMergedLogRecordScanner.java

```diff
@@ -185,16 +213,6 @@ public Builder withInstantRange(Option<InstantRange> instantRange) {
       return this;
     }
 
-    public Builder withLogRecordScannerCallback(LogRecordScannerCallback callback) {
-      this.callback = callback;
-      return this;
-    }
-
-    public Builder withRecordDeletionCallback(RecordDeletionCallback recordDeletionCallback) {
-      this.recordDeletionCallback = recordDeletionCallback;
```

Comment:
The `recordDeletionCallback` is only consumer for deleted reords from the delete avro block, we may need another iterator impl specifically for it.